admin/workflow_wiki/README.md

```diff
@@ -0,0 +1,421 @@
+          3 # with '#' will be ignored, and an empty message aborts the commit.
+          4 #
+          5 # On branch hotkey-docs
+          6 # Changes to be committed:
+          7 #   new file: simulators/nav/src/components/hotkeys.md
+          8 #
+          ```
+          ```console
+          $ git commit
+          [hotkey-docs 5c2d69ea] [nav-sim] Add hotkey documentation from HotKeys.vue
+           1 file changed, 33 insertions(+)
+           create mode 100644 simulators/nav/src/components/hotkeys.md
+          $ git status
+          On branch hotkey-docs
+          nothing to commit, working tree clean
+          ```
+        * I can type `git log` to view the change in the commit history. This is
+          one I like to do pretty often just like `git status`.
+          ```console
+          $ git log
+          ```
+          ```console
+          commit 5c2d69ea285843cc770fe44d99b064a54739db74 (HEAD -> hotkey-docs)
+          Author: Andrew Vernier <amverni@umich.edu>
+          Date:   Thu Nov 19 10:39:58 2020 -0500
+
+              [nav-sim] Add hotkey documentation from HotKeys.vue
+
+          commit 8dff03c3c4000fda2f2d4b990cdf5de2f24e4419 (upstream/main, origin/main, main)
+          ...
+          ```
+        * Lastly, I push my changes, so that I don’t lose any work if something were to happen to my local repository, using `git push origin hotkey-docs`.
+          ```console
+          $ git push origin hotkey-docs
+          Enumerating objects: 12, done.
+          Counting objects: 100% (12/12), done.
+          Delta compression using up to 8 threads
+          Compressing objects: 100% (7/7), done.
+          Writing objects: 100% (7/7), 1.20 KiB | 1.20 MiB/s, done.
+          Total 7 (delta 4), reused 0 (delta 0)
+          remote: Resolving deltas: 100% (4/4), completed with 4 local objects.
+          remote:
+          remote: Create a pull request for 'hotkey-docs' on GitHub by visiting:
+          remote:      http://github.com/amverni/mrover-workspace/pull/new/hotkey-docs
+          remote:
+          To github.com:amverni/mrover-workspace.git
+           * [new branch]        hotkey-docs -> hotkey-docs
+          ```
+
+    * Second work session
+        * If not already on my new branch (`hotkey-docs`), I check it out.
+        * Next, I make sure I am working off of the most up to date version of `upstream/main`.
+          I do this with `git pull --rebase upstream main`.
+          Note that most likely this will have no effect since I just created this branch off the most up to date version of `upstream/main`.
+          However, let's say I made a few code changes before `pull`ing.
+          In this case, I could `stash` them, then `pull --rebase`, and then, if this changes are for [issue 431](https://github.com/umrover/mrover-workspace/issues/431), I un`stash` (`stash pop`) them.
+          As the error message states, I could also commit them if I wanted.
```

Comment:
NIT: the styling on "un`stash`" is visually confusing here. I would just say "unstash"